src/Illuminate/Contracts/Queue/Queue.php

```diff
@@ -96,4 +96,19 @@ public function getConnectionName();
      * @return $this
      */
     public function setConnectionName($name);
+
+    /**
+     * Set the queue prefix.
+     *
+     * @param  string  $prefix
+     * @return $this
+     */
+    public function setQueuePrefix($prefix = null);
+
+    /**
+     * Get the queue prefix.
+     *
+     * @return string
+     */
+    public function getQueuePrefix();
```

Comment:
>  Is the simple solution to remove it from the contract for 5.4? as I stated no one is required to use the prefix, so we'd be fine without it.

Yes it should not be part of the contract in `5.4`, and only do so in `5.5`. That how it being done before and should remain so.